tests/unit/test_orchestrator.py

```
@@ -556,3 +559,103 @@ async def test_dispose_calls_task_registry_cancel_all(self):
+        lifecycle = _make_mock_lifecycle_manager()
+        batch_processor = AsyncMock()
+        batch_processor.process = AsyncMock(return_value=True)
+        batch_processor.estimate_chunks = AsyncMock(return_value=1)
+
+        orch, *_ = await _build_orchestrator(
+            task_registry=task_registry,
+            lifecycle_manager=lifecycle,
+            batch_processor=batch_processor,
+        )
+
+        # session_flush を実行（バックグラウンドタスクが登録される）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)